#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_uptodata_configure.dpatch by Paul Brossier <piem@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: this patch allows to run up-to-date configure scripts
## DP: see /usr/share/doc/autotools-dev/README.Debian.gz            

@DPATCH@
--- a/config.sub
+++ b/config.sub
@@ -1,4 +1,8 @@
 #! /bin/sh
+if [ -x /usr/share/misc/config.sub ]; then
+    exec /usr/share/misc/config.sub "$@"
+fi
+
 # Configuration validation subroutine script.
 #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
 #   2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008
--- a/config.guess
+++ b/config.guess
@@ -1,4 +1,8 @@
 #! /bin/sh
+if [ -x /usr/share/misc/config.guess ]; then
+    exec /usr/share/misc/config.guess "$@"
+fi
+
 # Attempt to guess a canonical system name.
 #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
 #   2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008
